Security/cve-2024-27982-http-request-smuggling/main.fmf

```diff
@@ -0,0 +1,14 @@
+summary: CVE-2024-27982 test
+description: Test for CVE-2024-27982
+contact: Patrik Koncity <pkoncity@redhat.com>
+test: ./runtest.sh
+recommend:
+- llhttp
+- rpm-build
+- dnf-plugins-core
+tag:
+  - CI-Tier-1
```

Comment:
I would remove Tier-1 tags, It won't be need to run it through every gating. WDYT?

Reply:
Removed.